sdk/include/ddk/ntpoapi.h

```diff
@@ -30,6 +30,13 @@ extern "C" {
 #ifndef _PO_DDK_
 #define _PO_DDK_
 
+typedef struct _SYSTEM_POWER_INFORMATION {
```

Comment:
Put this somewhere around PROCESSOR_STATE_HANDLER2 in L265/272 please.

Reply:
Done.